src/hotspot/cpu/s390/vm_version_s390.hpp

```diff
@@ -88,6 +88,9 @@ class VM_Version: public Abstract_VM_Version {
 // z13 end
 #define  MiscInstrExt2Mask              0x0000000000000020UL  // z14
 #define  MiscInstrExt3Mask              0x0000000000000004UL  // z15
+
+#define BEAREnhFacilityMask             0x4000000000000000UL  // z16, BEAR-enhancement facility, Bit: 193
+#define NNPAssistFacilityMask           0x0000000000040000UL  // z16, Neural-network-processing-assist facility, Bit: 165
```

Comment:
[My script](https://github.com/offamitkumar/s390xFacilityMask) says that this should be 
```suggestion
#define NNPAssistFacilityMask           0x0000000004000000UL  // z16, Neural-network-processing-assist facility, Bit: 165
```

```bash
Enter Facility Bit Number: 165
_feature array index: 2
bit position to set: 37
bit string: 0000000000000000000000000000000000000100000000000000000000000000
Mask:  0x0000000004000000UL
```